### PR TITLE
[housekeeping] add demo pitch

### PR DESCRIPTION
**Pull request by wattsecs**
Add a markdown file containing the demonstration pitch. Every subsequent demo should base itself on it and improve it such as to have a good pitch ready for the final demonstration.

### CODE REVIEW

**Review by NicolasVial — APPROVED** [review]
Good for now, Thank you (Do not merge it now please, I will modify some stuff for this week demo :) )

File: hl_pitch.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+_goes and looks at dataset overview_
+
+But there are only very few images... I want to have more objects to match! This is why I am going to go and augment my dataset.
+
+_adds picture and shows it added afterwards_
+
+Now let's learn some abstraction!
+
+_goes and learns in representation mode_
```
Comment (Nitwix) [inline hl_pitch.md]: Shouldn't we put the learning before the editing, to first showcase the main feature of the app?

Comment (wattsecs) [reply hl_pitch.md]: I decided the take the approach "first let's look what I can learn on" but one could argue that "I want to learn, oh my experience was not that good let's add more images" might actually make more sense

Comment (bbjubjub2494) [reply hl_pitch.md]: my iteration of the demo was criticised with this exact angle

Comment (NicolasVial) [inline hl_pitch.md]:
```suggestion
So let's learn!

Let's see what datasets are available to me...

_goes and looks at datasets overview (In the learning part)_ 

_goes in representation mode and start learning and remark that a picture shouldn't be in this dataset_

 This picture shouldn't be in this dataset and I don't have enough pictures to learn well... Let's see what is going on.
 
 _goes in datasets overview, select the good dataset, first add a new picture from the storage, then go check to see if the picture has been added. There is also the picture that should not be in this dataset, delete it._
 
 Nice, now it is perfect ! I can go back to learning with my improved dataset.
```

**Review by NicolasVial — COMMENTED** [review]
What do you guys think about this idea ? open to any comment :)

File: hl_pitch.md (hunk 1 above)
Comment (wattsecs) [reply hl_pitch.md]: `Let's select the representation mode...` is not necessary imo. We can see that you do it.

Comment (NicolasVial) [reply hl_pitch.md]: You're right :)

**Review by bbjubjub2494 — APPROVED** [review]

**Pull request merged**